db/migrate/20220804193922_add_new_fields_for_encrypted_intake_attributes_to_intake.rb

```diff
@@ -0,0 +1,15 @@
+class AddNewFieldsForEncryptedIntakeAttributesToIntake < ActiveRecord::Migration[7.0]
+  def change
+    add_column :intakes, :primary_last_four_ssn, :text
+    add_column :intakes, :spouse_last_four_ssn, :text
+    add_column :intakes, :primary_ssn, :text
+    add_column :intakes, :spouse_ssn, :text
+    add_column :intakes, :bank_account_number, :text
+    add_column :intakes, :bank_name, :string
```

Comment:
string for the ones that will no longer be encrypted, text for the rails encrypts fields